doc/md_src_files/090_futureWork.md

```diff
@@ -2,16 +2,56 @@
 
 # Future Work
 
-We intend SSVC to offer a workable baseline from which to improve and refine a vulnerability-prioritization methodology. While the method herein should be functional, we do not claim it is ready for use as is. Therefore, we lay out some aspects of future work that would help make it ready to use. We focus on further requirements gathering, further testing of the reliability of the decision process, and expanding to additional types of stakeholders beyond deployers and suppliers.
+We intend SSVC to offer a workable baseline from which to improve and refine a vulnerability-prioritization methodology.
+We are working to improve SSVC.
+Several of the future work items in this section have [issues associated with them on the SSVC GitHub page](https://github.com/CERTCC/SSVC/issues), which is a good place to go to check on progress or help.  
+Plans for future work focus on further requirements gathering, analysis of types of risk, and further testing of the reliability of the decision process.
 
 ## Requirements Gathering via Sociological Research
 
 The community should know what users of a vulnerability prioritization system want. To explore their needs, it’s important to understand how people actually use CVSS and what they think it tells them. In general, such empirical, grounded evidence about what practitioners and decision makers want from vulnerability scoring is lacking. We have based this paper’s methodology on multiple decades of professional experience and myriad informal conversations with practitioners. Such evidence is not a bad place to start, but it does not lend itself to examination and validation by others. The purpose of understanding practitioner expectations is to inform what a vulnerability-prioritization methodology should actually provide by matching it to what people want or expect. The method this future work should take is long-form, structured interviews. We do not expect anyone to have access to enough consumers of CVSS to get statistically valid results out of a short survey, nor to pilot a long survey.
 
+Coordinators in particular are likely to be heterogeneous.
+While the FIRST service frameworks for PSIRTs and CSIRTs differentiate two broad classes of coordinators, we have focused on CSIRTs here.
+PSIRTs may have somewhat different concerns.
+Investigating the extent to which SSVC should be customized for this group is future work as well.
+
+## Types of Risks
+
+SSVC estimates the relative risk created by a vulnerability in an information system.
+The priority of acting to mitigate or remediate a vulnerability goes up as this vulnerability risk goes up.
+SSVC does not currently take into account the change risk due to applying a mitigation or remediation.
+
+One way to view what SSVC currently provides is that it tells you how urgently a stakeholder should analyze overall risk due to a vulnerability risk.
```

Comment:
should analyze overall risk due to a vulnerability risk.

change to: should analyze overall risk due to a vulnerability.